servlet/src/test/java/io/undertow/servlet/test/dispatcher/DispatcherIncludeTestCase.java

```diff
@@ -258,9 +267,28 @@ public void testParametersAreMerged() throws IOException {
             HttpGet get = new HttpGet(DefaultServer.getDefaultServerURL() + "/servletContext/dispatch?param1=v11&param1=v12");
             get.setHeader("include", "/echo-parameters?param1=v13&param1=v14");
             HttpResponse result = client.execute(get);
-            Assert.assertEquals(StatusCodes.OK, result.getStatusLine().getStatusCode());
+            assertEquals(StatusCodes.OK, result.getStatusLine().getStatusCode());
             String response = HttpClientUtils.readResponse(result);
-            Assert.assertEquals(IncludeServlet.MESSAGE + "param1='v13,v14,v11,v12'", response);
+            assertEquals(IncludeServlet.MESSAGE + "param1='v13,v14,v11,v12'", response);
+        } finally {
+            client.getConnectionManager().shutdown();
+        }
+    }
+    @Test
+    public void testDisptacherServletInclude() throws IOException, InterruptedException {
+        TestHttpClient client = new TestHttpClient();
+        try {
+            HttpGet get = new HttpGet(DefaultServer.getDefaultServerURL() + "/servletContext/dispatchServletInclude");
+            HttpResponse result = client.execute(get);
+            assertEquals(StatusCodes.OK, result.getStatusLine().getStatusCode());
+            final String response = HttpClientUtils.readResponse(result);
+            //UNDERTOW-2245 javax.servlet.forward.mapping request attribute is not available in servlets forwarded with RequestDispatcher
+
+            assertTrue(DeploymentUtils.isContainString(response,"jakarta.servlet.include.context_path"));
+            assertTrue(DeploymentUtils.isContainString(response,"jakarta.servlet.include.servlet_path"));
+            assertTrue(DeploymentUtils.isContainString(response,"jakarta.servlet.include.request_uri"));
+            assertTrue(DeploymentUtils.isContainString(response,"jakarta.servlet.include.mapping"));
```

Comment:
missing path info and query string

Reply:
Hi @fl4via As discussed this is causing test failure, Spec need to review for this to fix.